### ISSUE TITLE
MAINTAINER deprecated

### ISSUE THREAD

**Issue by wkusnierczyk** [replicatedhq/dockerfilelint#71]
The `MAINTAINER` instruction is deprecated:

* https://docs.docker.com/engine/reference/builder/#/maintainer-deprecated

Consider implementing a warning with suggestion to replace with `LABEL maintainer`.

**Comment by jadametz** [comment]
Looks like this was solved in #69

**Comment by thomasbiddle** [comment]
@jadametz 

Confirmed - just ran whatever the latest is:
```
Line 4: MAINTAINER "some@email.com"
Issue  Category      Title                 Description
    1  Deprecation   Deprecated as of      This INSTRUCTION is deprecated as of Docker 1.13
                     Docker 1.13
```

This can be closed.

**Comment by nexdrew** [comment]
Thanks everyone! Not sure why this was left open, closing now.

**Comment by dserodio** [comment]
I added a MAINTAINER line to the www.fromlatest.io example and it didn't complain
